src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_ExcelCompat_PowerFxV1CompatDisabled.txt

```diff
@@ -965,4 +965,179 @@ Error({Kind:ErrorKind.InvalidArgument})
 "'long1ateti1e'"
 
 >> Text(Date(1970, 1, 1), DateTimeFormat.LongDateTime24)
-"'long1ateti1e24'"
\ No newline at end of file
+"'long1ateti1e24'"
+
+// Block exponential notation with decimals, escaping character and scaling modifier
```

Comment:
We don't need to add more tests to this file. Power Fx in C# never runs with V1 disabled, and the difference in the Text function is mostly a runtime difference. Those are interesting tests to be running in Power Apps, but we don't need them here.

Reply:
Removed the change in this file. thanks